### PR TITLE
Load commands automatically

### PR DESCRIPTION
**Pull request by robinkanters**

### CODE REVIEW

File: TrueCraft/Program.cs [hunk 1]
```diff
@@ -59,8 +59,7 @@ public static void Main(string[] args)
             }
             catch
             {
-                world = new World("default", new StandardGenerator());
-                world.BlockRepository = Server.BlockRepository;
+                world = new World("default", new StandardGenerator()) {BlockRepository = Server.BlockRepository};
```
Comment (SirCmpwn) [inline TrueCraft/Program.cs]: I don't like this change, please reverse it.

**Comment by SirCmpwn** [comment]
Looks good save for that one unrelated change in Program.cs.

File: TrueCraft/Program.cs (hunk 1 above)
Comment (robinkanters) [inline TrueCraft/Program.cs]: Yeah that's something that R# likes to do, have to say I'm neutral on this

**Comment by robinkanters** [comment]
have fun

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**